### PR TITLE
User's Address added

### LINKED ISSUE #1: User address field missing
User address field missing

### PR DESCRIPTION
**Pull request by ankitsaxenavivo**
User's Address added

### CODE REVIEW

**Review by sharmaVineet00 — CHANGES_REQUESTED** [review]
put comments.

**Review by ankitsaxenavivo — COMMENTED** [review]
done.

File: ConsoleApp1/ConsoleApp1/User.cs [hunk 1]
```diff
@@ -8,6 +8,10 @@ class User
     {
         public string Name { get; set; }
```
Comment (sharmaVineet00) [inline ConsoleApp1/ConsoleApp1/User.cs:9]: comment missing.

**Review by sharmaVineet00 — APPROVED** [review]

**Comment by sharmaVineet00** [comment]
good job.

**Pull request merged**